lib/genome/online_updaters/oncokb/updater.rb

```diff
@@ -0,0 +1,133 @@
+module Genome; module OnlineUpdaters; module Oncokb;
+  class Updater
+    attr_reader :new_version
+    def initialize(source_db_version = Date.today.strftime("%d-%B-%Y"))
+      @new_version = source_db_version
+    end
+
+    def update
+      remove_existing_source
+      source = create_new_source
+      create_interaction_claims(source)
```

Comment:
This is functionally valid. I'm curious why we're capturing and passing `source` here, though, when we have the `@source` instance variable defined at [Line 20](https://github.com/griffithlab/dgi-db/pull/233/files/1d3b361aad1a33dba44ed4a46b9f35de874bcece#diff-45d3567762a2cf334d0fdd1d1ee887d7R20)?